pkg/operator/targetconfigcontroller/targetconfigcontroller.go

```diff
@@ -179,21 +185,24 @@ func loglevelToZap(logLevel operatorv1.LogLevel) string {
 }
 
 func (c *TargetConfigController) getSubstitutionReplacer(operatorSpec *operatorv1.StaticPodOperatorSpec,
-	imagePullSpec, operatorImagePullSpec string, envVarMap map[string]string) (*strings.Replacer, error) {
+	imagePullSpec, operatorImagePullSpec string, envVarMap map[string]string, backupVars *backuphelpers.BackupConfig) (*strings.Replacer, error) {
 	var envVarLines []string
 	for _, k := range sets.StringKeySet(envVarMap).List() {
 		v := envVarMap[k]
 		envVarLines = append(envVarLines, fmt.Sprintf("      - name: %q", k))
 		envVarLines = append(envVarLines, fmt.Sprintf("        value: %q", v))
 	}
 
+	backVarLines := backupVars.ArgString()
+
 	return strings.NewReplacer(
 		"${IMAGE}", imagePullSpec,
 		"${OPERATOR_IMAGE}", operatorImagePullSpec,
 		"${VERBOSITY}", loglevelToZap(operatorSpec.LogLevel),
 		"${LISTEN_ON_ALL_IPS}", "0.0.0.0", // TODO this needs updating to detect ipv6-ness
 		"${LOCALHOST_IP}", "127.0.0.1", // TODO this needs updating to detect ipv6-ness
 		"${COMPUTED_ENV_VARS}", strings.Join(envVarLines, "\n"), // lacks beauty, but it works
+		"${COMPUTED_BACKUP_VARS}", backVarLines,
```

Comment:
just directly call it

```suggestion
		"${COMPUTED_BACKUP_VARS}", backupVars.ArgString(),
```

Reply:
done